sdk/math/storage/fee_tier.rs

```diff
@@ -0,0 +1,36 @@
+use crate::alloc::string::ToString;
+use crate::errors::InvariantError;
+use crate::types::percentage::Percentage;
+use crate::{convert, resolve};
+use decimal::*;
+use serde::{Deserialize, Serialize};
+use tsify::Tsify;
+use wasm_bindgen::prelude::*;
+
+#[derive(Default, Debug, Copy, Clone, PartialEq, Serialize, Deserialize, Tsify)]
+#[tsify(into_wasm_abi, from_wasm_abi)]
+pub struct FeeTier {
+    pub fee: Percentage,
+    pub tick_spacing: u16,
```

Comment:
We convert all int/uint values to `bigInt`, avoiding the conversion of integral values to ts `number` type